Doc/library/functions.rst

```diff
@@ -2018,6 +2018,9 @@ are always available.  They are listed here in alphabetical order.
    object, the names to import are retrieved and assigned to their respective
    names.
 
+   An :exc:`ImportError` exception can be raised if the module cannot be imported
+   successfully.
```

Comment:
I'm not sure I like this wording - what do you mean "can be raised", it sounds like it is possible for import to fail and not raise an exception.

Reply:
> Note Ronald's comment on arbitrary exceptions being raised from a module body while it's being imported.

It means arbitrary exceptions can also be raised.

Reply:
I understand the intent, but I don't think that's the only possible interpretation of the current wording.

Reply:
I just found out that https://github.com/python/cpython/pull/94662 is also doing the same thing. Would you prefer that one?